src/main/java/com/techiteasy/demo/dto/TelevisionDto.java

```diff
@@ -1,71 +1,89 @@
-        private Long id;
-        private String type;
-        private String brand;
-        private String name;
-        private Double price;
-        private LocalDate purchaseDate; //Datum voor inkoop
-        private LocalDate saleDate; //Datum voor verkoop
-        private Double availableSize; //Mogelijk een enum als je dit als optie wilt
-        private Integer refreshRate; //Mogelijk een enum als je dit als optie wilt
-        private String screenType; //Enum voor schermtype
-        private String screenQuality; //Enum voor schermkwaliteit
-        private Boolean smartTv;
-        private Boolean wifi;
-        private Boolean voiceControl;
-        private Boolean hdr;
-        private Boolean bluetooth;
-        private Boolean ambilight;
-        private Integer originalStock;
-        private Integer sold;
-
-        //Constructor met parameters
-        public TelevisionDto(Long id, String type, String brand, String name, Double price, LocalDate purchaseDate,
-                             LocalDate saleDate, Double availableSize, Integer refreshRate, String screenType,
-                             String screenQuality, Boolean smartTv, Boolean wifi, Boolean voiceControl,
-                             Boolean hdr, Boolean bluetooth, Boolean ambilight, Integer originalStock, Integer sold) {
-
-            this.id = id;
-            this.type = type;
-            this.brand = brand;
-            this.name = name;
-            this.price = price;
-            this.purchaseDate = purchaseDate;
-            this.saleDate = saleDate;
-            this.availableSize = availableSize;
-            this.refreshRate = refreshRate;
-            this.screenType = screenType;
-            this.screenQuality = screenQuality;
-            this.smartTv = smartTv;
-            this.wifi = wifi;
-            this.voiceControl = voiceControl;
-            this.hdr = hdr;
-            this.bluetooth = bluetooth;
-            this.ambilight = ambilight;
-            this.originalStock = originalStock;
-            this.sold = sold;
-        }
+    private Long id;
+    private String type;
+    private String brand;
+    private String name;
+    private Double price;
+    private LocalDate purchaseDate; //Datum voor inkoop
+    private LocalDate saleDate; //Datum voor verkoop
+    private Double availableSize; //Mogelijk een enum als je dit als optie wilt
+    private Integer refreshRate; //Mogelijk een enum als je dit als optie wilt
+    private String screenType; //Enum voor schermtype
+    private String screenQuality; //Enum voor schermkwaliteit
+    private Boolean smartTv;
+    private Boolean wifi;
+    private Boolean voiceControl;
+    private Boolean hdr;
+    private Boolean bluetooth;
+    private Boolean ambilight;
+    private Integer originalStock;
+    private Integer sold;
+    private RemoteControllerDto remoteController; //Toevoeging van RemoteControllerDto
+    private List<CIModuleDto> ciModules; //Toevoeging van CIModuleDto
```

Comment:
Dit is prima, maar in plaats van "List<CiModuleDto>" kun je hier ook "List<Long>" gebruiken. Dan zet je enkel de id's van de cimodules er in. Dit is vooral handig wanneer je problemen krijgt met recursie.